### PR TITLE
AirQloud  based data summary reports (calibrated data availabilty for customisable time period)

### PR DESCRIPTION
**Pull request by sserurich**
#### Summary of Changes (What does this PR do?)

-  Added section on airqloud overview page for generating AirQloud based data summary reports

#### Status of maturity (all need to be checked before merging):

- [x ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state

#### How should this be manually tested?
 -  Fetch branch locally
 -  run npm run stage

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
image

### CODE REVIEW

File: netmanager/src/views/components/AirQlouds/AirQloudDataReport.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <Paper className={classes.root}>
+      <h2>Report Title</h2>
+      <p>This is the content of the report.</p>
+    </Paper>
+  );
+}
+
+export default AirQloudDataReport;
```
Comment (Codebmk) [inline netmanager/src/views/components/AirQlouds/AirQloudDataReport.js:24]: Thanks @sserurich . The file called `AirQloudDataReport` doesn't seem to be used anywhere

**Comment by Codebmk** [comment]
@sserurich I have added an empty state for reports and adjusted content titles.
image
image

**Review by Baalmart — APPROVED** [review]
this is amazing! Thanks @sserurich !

After @Codebmk  approves changes, I will move ahead and merge the PR.

**Comment by PaulZana** [comment]
@sserurich thanks for your efforts! This is gonna be handy for the screens I'm currently working on.

File: netmanager/src/views/components/AirQlouds/AirQloudDataReport.js (hunk 1 above)
Comment (sserurich) [reply netmanager/src/views/components/AirQlouds/AirQloudDataReport.js]: @Codebmk , I have removed this and made few updates, kindly  test this out.

**Review by Codebmk — COMMENTED** [review]
@sserurich airqlouds summary table is empty. Could you be knowing the cause?
image
image

**Comment by sserurich** [comment]
> @sserurich airqlouds summary table is empty. Could you be knowing the cause? image image

@Codebmk , I can't reproduce this, kindly share some more info, according to the screenshot, it shows airqlouds aren't listed, and I can't get this scenario reproduced(from looking at how this was designed when the data isn't there that's when that message shows up no data to display,  as an exampleI have checked with an actual airqloud which had no sites (and that's the message displayed). However whenever I open the airqloud registry, I always get that list populated.. cc @NoahNsimbe , just to confirm whether

**Comment by Codebmk** [comment]
@sserurich It could have been on the infra side. It is no longer there.

File: netmanager/src/config/urls/deviceMonitoring.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { stripTrailingSlash } from "../utils";
 
 const BASE_DEVICE_MONITORING_URL = stripTrailingSlash(
   process.env.REACT_APP_BASE_DEVICE_MONITORING_URL ||
-    process.env.REACT_APP_BASE_URL
+    process.env.REACT_APP_BASE_URL_V2
 );
```
Comment (Codebmk) [inline netmanager/src/config/urls/deviceMonitoring.js:7]: You've changed the device monitoring url to v2 hence the errors in the network monitoring pages. Could you only use this v2 env variable on the api route?
image

File: netmanager/package-lock.json
Comment (Codebmk) [inline netmanager/package-lock.json:1]: The lock file got updated with no corresponding update in the package.json which could cause problems... @123MwanjeMike  could confirm?

**Review by Codebmk — APPROVED** [review]
LGTM

**Review by Baalmart — CHANGES_REQUESTED** [review]
Hi @Codebmk , let us remove this lock file first.
Change is fishy and might cause us problems
cc: @sserurich

**Pull request closed without merging**